packages/editor/src/plugins/text/hooks/use-editable-key-down-handler.tsx

```diff
@@ -41,11 +48,38 @@ export const useEditableKeydownHandler = (
       }
 
       // Handle specific keyboard commands
-      // (only if selection is collapsed and suggestions are not shown)
+      // (only if selection is collapsed)
       const { selection } = editor
-      if (selection && Range.isCollapsed(selection) && !showSuggestions) {
+      if (selection && Range.isCollapsed(selection)) {
         const isListActive = isSelectionWithinList(editor)
 
+        if (event.key === '/') {
+          const { path } = selection.focus
+          const node = Node.get(editor, path)
+
+          const parent = selectChildTreeOfParent(store.getState(), id)
+
+          if (Object.hasOwn(node, 'text') && node.text.length === 0 && parent) {
+            pluginMenuDispatch({
+              type: PluginMenuActionTypes.OPEN_WITH_SLASH_KEY,
+              payload: {
+                insertIndex: parent.children?.findIndex(
+                  (child) => child.id === id
+                ),
+                onInsertComplete: () => {
```

Comment:
@hejtful I think this is undesired behaviour, perhaps we can discuss this on a call? 

I introduced the callback for a different reason, after your changes it was unnecessary for me to use it that way.

An example of how this is broken is seen with the failed test in 423-text-plugin-list.ts

Reply:
Removing the onInsertComplete callback shows that the insertIndex is incorrect, and the new plugin gets inserted before the existing one

Reply:
I have added a change (hopefully fix) in `packages/editor/src/plugins/text/hooks/use-editable-key-down-handler.tsx: 66`

Reply:
Could we maybe simplify this by handling Click and Slash behaviour in the same way?
I think removing the current text-plugin if it's empty would work nicely for both.

Reply:
Open with slash key is opening the Modal from within a text plugin, while normal open is clicking the Floating add button, which to me as a User I would expect to add an additional row

Reply:
The way I understood it:
- Click on the plus button between two rows:
  1. Open the modal
  2. Select the plugin
  3. Insert the selected plugin where the button was
- Slash in an empty Text plugin:
  1. Open the modal
  2. Select the plugin
  3. Insert the selected plugin after the empty Text plugin
  4. Remove the empty Text plugin

@elbotho Are we aligned here? If yes, could you please make a more concrete suggestion about handling both in the same way / removing the current text plugin for both?

Reply:
but I would also not mind to replace an empty text plugin. Currently the only way to add a new plugin in place of the currently selected empty text plugin is to add it and the manually remove the text plugin.

Reply:
I'm talking about this case:

- Click on the plus button between two rows when the button is below an empty text plugin:
  1. Open the modal
  2. Select the plugin
  3. Insert the selected plugin after the empty Text plugin
  4. Remove the empty Text plugin

Which would be the steps as `Slash in an empty Text plugin` so that's why I wondered if we could simplify:

``` diff
   OPEN = 'OPEN',
-  OPEN_WITH_SLASH_KEY = 'OPEN_WITH_SLASH_KEY',
   CLOSE = 'CLOSE',
  ```

Reply:
I think @GregorZupan should have some input here, I don't agree that i makes sense to remove the empty text plugin if they clicked on the button